src/Tests/Http/ResponseTest.php

```diff
@@ -32,6 +32,7 @@ public function testGettingAndSettingBody(): void
         $body1 = $this->createMock(IHttpBody::class);
         $response = new Response(200, null, $body1);
         $this->assertSame($body1, $response->getBody());
+        $this->assertSame("HTTP/1.1 200 OK\r\n\r\n", (string)$response);
```

Comment:
I'm not sure we need to test this because it's outside the scope of actually setting the body. Also, the mocked body isn't set up to return anything when serialized, which means we aren't really testing anything by serializing the response.

Reply:
Hi @opulencephp, thank you for your reply.
You're right. We don't have to add this assertion because we just test the mocked class, not the actual class.

I will remove them.

Reply:
I've removed this unused assertion, please check that :).